ui-tests/src/test/resources/features/integrations/service-now.feature

```diff
@@ -0,0 +1,76 @@
+@servicenow
+Feature: Integration - ServiceNow-amq/log
+
+  Background: Clean application state
+    Given deploy AMQ broker and add accounts
+    Given clean application state
+    Given log into the Syndesis
+    Given clean destination type "queue" with name "incidents"
+    Given clean "TODO" table
+    Given created connections
+      | ServiceNow         | Servicenow | ServiceNow | Service-Now connection |
+      | AMQ Message Broker | AMQ        | AMQ        | AMQ connection         |
+
+
+  Scenario: Get incident
+    # create integration
+    When navigate to the "Home" page
+    And click on the "Create Integration" button to create a new integration.
+    Then check visibility of visual integration editor
+
+    And check that position of connection to fill is "Start"
+
+    When select the "ServiceNow" connection
+    And select "Retrieve Record" integration action
+    And .*fill in values
+      | Limit of elements per page              | 1                |
+      | The query used to filter the result set | number=QACUSTOM4 |
+    And select "Incident" from "table" dropdown
+    And click on the "Done" button
+    Then check that position of connection to fill is "Finish"
+
+    When select the "AMQ" connection
+    And select "Publish Messages" integration action
+
+    And fill in values
+      | Destination Name | incidents |
+      | Destination Type | Queue     |
+    And click on the "Next" button
+    And click on the "Done" button
+
+    When click on the "Publish" button
+    And set integration name "service-now-2-amq"
+    And click on the "Publish" button
+    And create incident with "QACUSTOM4" number
+    And wait for integration with name: "service-now-2-amq" to become active
+    Then verify that received incident from "incidents" queue contains "QACUSTOM4"
+
+    # create incident
+  Scenario: Create incident using
+    When navigate to the "Home" page
+    And click on the "Create Integration" button to create a new integration.
+    Then check visibility of visual integration editor
+
+    And check that position of connection to fill is "Start"
+
+    When select the "AMQ" connection
+    And select "Subscribe for messages" integration action
+
+    And fill in values
+      | Destination Name | incidents-create |
+      | Destination Type | Queue            |
+    And click on the "Next" button
+    And click on the "Done" button
+    Then check that position of connection to fill is "Finish"
+
+    When select the "ServiceNow" connection
+    And select "Create Record" integration action
+    And select "Add new Incident" from "table" dropdown
+    And click on the "Next" button
+    When click on the "Publish" button
+    And set integration name "amq-2-snow"
+    And click on the "Publish" button
+    And wait for integration with name: "amq-2-snow" to become active
```

Comment:
This is rest step to check if integration works, in UI we use different approach, check other feature files. But this should work anyway as there is 2 min. sleep if on jenkins...